core/src/Streamly/Internal/FileSystem/Handle.hs

```diff
@@ -124,25 +124,23 @@ import Streamly.Internal.Data.Unfold.Type (Unfold(..))
 import Streamly.Internal.Data.Array.Unboxed.Type
        (Array(..), writeNUnsafe, unsafeFreezeWithShrink, byteLength)
 import Streamly.Internal.Data.Stream.Type (Stream)
-import Streamly.Internal.Data.Stream.IsStream.Type
-    (IsStream, mkStream, fromStreamD)
-import Streamly.Internal.Data.Array.Stream.Foreign (lpackArraysChunksOf)
+import Streamly.Internal.Data.Stream (fromStreamD)
+-- import Streamly.Internal.Data.Array.Unboxed (lpackArraysChunksOf)
 -- import Streamly.String (encodeUtf8, decodeUtf8, foldLines)
 import Streamly.Internal.System.IO (defaultChunkSize)
 
 import qualified Streamly.Data.Fold as FL
 import qualified Streamly.Data.Array.Unboxed as A
 import qualified Streamly.Internal.Data.Array.Unboxed.Type as A
-    (byteLength, asPtrUnsafe)
-import qualified Streamly.Internal.Data.Array.Unboxed.Mut as MArray
-    (Array(..), newPinnedArrayBytes, asPtrUnsafe)
-import qualified Streamly.Internal.Data.Array.Stream.Foreign as AS
+import qualified Streamly.Internal.Data.Array.Unboxed.Mut.Type as MArray
+--import qualified Streamly.Internal.Data.Array.Unboxed as AS
+import qualified Streamly.Internal.Data.Array.Stream.Mut.Foreign as AS
```

Comment:
Let's move `Streamly.Internal.Data.Array.Stream.Foreign` to `streamly-core` in a separate PR, and then base this PR on that one.